### PR TITLE
#254 german translation contributor chapter 02

### PR DESCRIPTION
**Pull request by RJPlog**
Hi, I again found some time to translate another chapter of the contributor series into german, chapter 02. Letters are swirling around my eyes now and reading the 5th time will not change it anymore, so I will set this for review.

### CODE REVIEW

**Comment by rrrutledge** [comment]
Very exciting, @RJPlog!  Looking forward to the review and merge!

**Review by lenucksi — APPROVED** [review]
Thanks @RJPlog for another great translation! 🎉 
I've found a typo which I'll add in as commit suggestion and once sentence where I wasn't sure if there wouldn't be a slightly different way to write it. 

I'd be great if @arnom-ms could have a second look - and also find all the comma issues if there are any. 😉

File: contributor/de/02-becoming-a-contributor-article-de.asciidoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Ein guter Contributor kann einfach beim Host Team anrufen und, wenn es sowohl technisch als auch geschäftlich sinnvoll ist, eine Abhängigkeit und Wiederverwendung einer Komponente anzustreben, anstatt die Arbeit zu duplizieren. Contibutoren können auch dem Management die Vorteile von InnerSource Beiträgen erklären.
+
+
+=== Der Umfang von InnerSource Beiträgen
+
+Geht es bei Inner__Source__ nur um __Source__Code? Natürlich nicht. Falls die Aufgabe Deines Teams von äußeren Komponenten abhängt, möchtest Du sicher sein, dass diese gut gewartet sind und funktionieren. Als ein InnerSource Contributor kannst Du das Host Team auf viele Arten unterstützen. Das Melden von Fehlern oder Auffälligkeiten beim Benutzen der Komponenten ist ein wertvoller Beitrag. Ebenso das Erstellen oder das Fixen von Testfällen, die zeigen das der Code nicht funktioniert wie erwartet. Auch die Verbesserung der Dokumentation, damit andere diese besser verstehen und dazu beitragen können. Auch andere zu unterstützen oder Hilfe beim Fixen von Bugs können wertvolle Beiträge sein. Ein weiteres Beispiel für einen wertvollen Beitrag ist die Verbesserung von Builds.
```
[the remark this answers is not in the document]
Comment (RJPlog) [reply contributor/de/02-becoming-a-contributor-article-de.asciidoc:33]: Hi Johannes, thanks for your review and doing the corrections directly as a commit - so meanwhile I can start with chapter 03. ;-)

**Comment by tsadler1988** [comment]
Thanks @RJPlog and @lenucksi!

**Comment by rrrutledge** [comment]
Nice work, all!

**Pull request merged**